packages/sync-actions/test/product-types-sync-base.spec.js

```diff
@@ -4,7 +4,7 @@ import { baseActionsList } from '../src/product-types-actions'
 
 describe('Exports', () => {
   test('action group list', () => {
-    expect(actionGroups).toEqual(['base', 'attributes'])
+    expect(actionGroups).toEqual(['base'])
```

Comment:
why removing `'attributes'` here?

Reply:
because attributes should not be part of the `actionGroups` which I assume are the target keys for `doMapActions`

https://github.com/commercetools/nodejs/blob/13f0f3a0a2343d0852011ecf71418f93172a16af/packages/sync-actions/src/product-types.js#L37-L47